src/view/style-marshal/get-styles.js

```diff
@@ -113,8 +114,8 @@ export default (styleContext: string): Styles => {
   // > Applied while the user is actively dragging
 
   // cursor: grab
-  // We apply this by default for an improved user experience. It is such a common default that we
-  // bake it right in. Consumers can opt out of this by adding a selector with higher specificity
+  // We apply apply this style to the body in case the user pointer
```

Comment:
absolute nitpick but you've written 'apply' here twice

Reply:
i'll fix it

Reply:
fixed